src/routes/home/Home.js

```diff
@@ -431,6 +449,29 @@ class Home extends React.Component {
       formatted_address: 'Finding Address...',
       addressProvenance,
     });
+
+    // show error message if location is outside NYC
+    console.time('new PolygonLookup'); // eslint-disable-line no-console
+    const lookup = new PolygonLookup(
+      this.props.boroughBoundariesFeatureCollection,
+    );
+    console.timeEnd('new PolygonLookup'); // eslint-disable-line no-console
+    const end = { latitude, longitude };
+    const BoroName = getBoroNameMemoized({ lookup, end });
```

Comment:
consider deduping this with ElectriCitibikes.js

Reply:
this won't really work since `ElectriCitibikes.js` builds the lookup a little while before using it, rather than immediately before